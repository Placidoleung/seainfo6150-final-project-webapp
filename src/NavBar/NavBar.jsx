import React from 'react'
import styles from './NavBar.module.css'

const NavBar = () => {

    return(
        <div className="container">
            <div className={styles.logo}>
                <a href="/">BlockNews</a>
            </div>
            <ul className={styles.nav}>
                <li className={styles.item}><a href="/">Home</a></li>
                <li className={styles.item}><a href="/contact">Contact</a></li>
                <li className={styles.item}><a href="/about">About</a></li>
            </ul>
        </div>

    );

}

export default NavBar;