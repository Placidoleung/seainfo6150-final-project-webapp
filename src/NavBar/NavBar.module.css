@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Orelega+One&family=Zen+Dots&display=swap');

.logo {
    display: flex;
    justify-content: center;
    align-items: center;
    
}

.logo a{
    color: black;
    text-decoration: none;
    font-family: 'Zen Dots', cursive;
    font-size: xx-large;
}

.nav {
    list-style-type: none;
    margin: 0;
    padding: 0;
    overflow: hidden;
    background-color: #333;
}
 
.item {
    float: left;
}
 
.item a {
    display: block;
    color: white;
    font-family: 'Zen Dots', cursive;
    text-align: center;
    padding: 14px 16px;
    text-decoration: none;
}
 
.item a:hover {
    background-color: #111;
}
