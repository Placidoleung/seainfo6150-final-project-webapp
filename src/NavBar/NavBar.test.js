import React from 'react'
import NavBar from './NavBar'

describe("NavBar tests", () => {

    it('render correctly ', () => {
        const { container } = render(<NavBar />);
        expect(container).toMatchSnapshot();
    });

});