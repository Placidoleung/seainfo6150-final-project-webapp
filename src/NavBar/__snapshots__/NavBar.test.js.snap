// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar tests render correctly  1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="logo"
    >
      <a
        href="/"
      >
        BlockNews
      </a>
    </div>
    <ul
      class="nav"
    >
      <li
        class="item"
      >
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li
        class="item"
      >
        <a
          href="/contact"
        >
          Contact
        </a>
      </li>
      <li
        class="item"
      >
        <a
          href="/about"
        >
          About
        </a>
      </li>
    </ul>
  </div>
</div>
`;
